templates/default_hostgroup.yaml.erb

```diff
@@ -0,0 +1,10 @@
+---
+:default_hostgroup:
+  :facts_map:
+  <%= @hostgroups.each do |group, fact_kv| -%>
```

Comment:
It'd a good idea to keep it sorted to avoid rewrites (same below)
```suggestion
  <%= @hostgroups.sort.each do |group, fact_kv| -%>
```

Reply:
I am big fan of sorted hash, I rushed this one.